src/robot/libraries/BuiltIn.py

```diff
@@ -2993,11 +2994,13 @@ def log(self, message, level='INFO', html=False, console=False,
         the ``html`` argument is using the HTML pseudo log level. It logs
         the message as HTML using the INFO level.
 
-        If the ``console`` argument is true, the message will be written to
-        the console where test execution was started from in addition to
-        the log file. This keyword always uses the standard output stream
-        and adds a newline after the written message. Use `Log To Console`
-        instead if either of these is undesirable,
+        If the ``console`` argument is true or the log level is ``CONSOLE``,
+        the message will be written to the console where test execution was
+        started from in addition to the log file. This keyword always uses the
+        standard output stream and adds a newline after the written message.
+        Use `Log To Console` instead if either of these is undesirable,
+    Mimic html section...
```

Comment:
I believe wrote that as a note during the online session and purposefully set wrong indent so that I would clean it up.
I'll try to be more thorough next time :grimacing: